ElevatorSim/src/elevatorsim/floor/FloorController.java

```diff
@@ -40,6 +31,7 @@ public FloorController(String name, int numOfFloors, Map<Integer, ElevatorReques
 		}
 
 		this.requests = requests;
+		floorEvents = new FloorEvents(this);
```

Comment:
this thread never starts

Reply:
true, i'll start the thread when the FloorController runs